sup3r/preprocessing/derivers/methods.py

```diff
@@ -402,6 +418,8 @@ def compute(cls, data):
     'cloud_mask': CloudMask,
     'clearsky_ratio': ClearSkyRatio,
     'sza': Sza,
+    'latitude_feature': LatitudeFeature,
```

Comment:
You can just do ``'latitude_feature': 'latitude'`` here. The data handler can already get 'latitude' through `dh.data` it just isn't seen in `dh.features` because ``dh.features`` is defined as ``list(dh.data.data_vars)`` and 'latitude' is in ``dh.data.coords``.

Reply:
oh nice, done here: https://github.com/NREL/sup3r/pull/250/commits/9f5dfcd800a120f0c27bdb0edb67d2821aa5b003